### DISCUSSION TITLE
Theme Manager - Programmable Theme Switching

### DISCUSSION THREAD

**Discussion by thaigrocer** [OnionUI/Onion#655] [Ideas]
I am inspired by [3DS theme manager](https://github.com/astronautlevel2/Anemone3DS) where multiple themes can be programmed to automatically change on a particular event(s). Some events include waking, going back to home screen, powering on, arbitrary timer, etc.
Popular uses include randomizing, rotating.

Why a theme manager would be good to have:
- Manager helps fully utilize available themes
There are many high quality themes made by the community that are underutilized
- Increases community engagement - greater utilization therefore appreciation, more themes created in turn

Thanks! Hopefully I didn't overlook a similar discussion in my searching. This is for sure low priority.

**Comment by NikitaRUniverse** [comment]


**Reply by Aemiii91** [reply]
This is not the right place to file a bug report - but, noted! 👌
